docs/src/tutorials/huggingface.rst

```diff
@@ -0,0 +1,183 @@
+From the config file, you can specify
+
+.. literalinclude:: /../../examples/huggingface/config.yaml
+  :language: yaml
+  :lines: 29-31
+
+This will change your working directory. You can see that with the hydra-orion-sweeper, we are able
+to specify 4 variables from Orion. ``${hydra.sweeper.orion.name}`` for the ``experiment_name``,
+``${hydra.sweeper.orion.id}`` for the ``experiment_id``, ``${hydra.sweeper.orion.uuid}`` for the
+experiment ``uuid`` and ``${hydra.sweeper.orion.trial}`` for the ``trial_id``.
+
+In the code, you can now specify the output directory to the trainer with the
+``output_dir`` parameter. ``os.getcwd()`` specifies the current working dir.
+
+.. code-block:: python
+
+    output_dir=str(os.getcwd())+"/test_trainer",
+
+You can find more about the Hydra-Orion-Sweeper plugin by looking directly at the
+Github Repository : `Hydra_Orion_Sweeper <https://github.com/Epistimio/hydra_orion_sweeper>`_ ,
+or find out more about Hydra in general here : `Hydra <https://hydra.cc/docs/intro/>`_
+
+
+Adapting the code for Hydra
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The ``config_path`` and ``config_name`` here specifies the path of your Hydra file.
+The ``cfg.args`` are reference to the ``args`` in the config file.
+We also are going to be using this main function as the entry point of the program.
+
+.. literalinclude:: /../../examples/huggingface/main.py
+  :language: python
+  :lines: 63-68, 229-230
+
+With the ``hydra-orion-sweeper``, this function needs to return the objective. You have 3 choices
+for what you can return :
+
+.. code-block:: python
+
+    if cfg.return_type == "float":
+        return out
+
+    if cfg.return_type == "dict":
+        return dict(name="objective", type="objective", value=out)
+
+    if cfg.return_type == "list":
+        return [dict(name="objective", type="objective", value=out)]
+
+For the purpose of this tutorial, we are going to keep it simple and simply return a float,
+our objective that we want to minimize.
+
+.. literalinclude:: /../../examples/huggingface/main.py
+  :language: python
+  :lines: 213
+
+Comet
+^^^^^
+We are going to use Comet to track our experiments. It is quite simple to use. First,
+install comet using
+
+.. code-block:: bash
+
+   $ pip install comet-ml
+
+Now that it is installed, we simply have to set some environment variables, such as
+
+.. literalinclude:: /../../examples/huggingface/main.py
+  :language: python
+  :lines: 7-10
+
+You can also set them in your working environment. If you are to set them in python, however,
+you need to make sure to set them before importing ``transformers``.
+
+For the ``COMET_API_KEY``, you will be given a token when creating your comet account.
+This is the one you are going to use here.
+
+And that is it ! If the variables are set and comet-ml is downloaded, HuggingFace will
+automatically upload your data to Comet, you simply have to go to your profile on their site
+and see your experiments.
```

Comment:
It would be good to add here that we can easily swap Comet logger to many others, including wandb, MLFlow, Neptune and ClearML: https://huggingface.co/docs/transformers/main_classes/callback#callbacks.

Reply:
Added in [4a5965d](https://github.com/Epistimio/orion/pull/1040/commits/4a5965dc093261b1747a1a06d75cd67970f9262a)